sql/core/src/test/resources/sql-tests/inputs/string-functions.sql

```diff
@@ -231,3 +231,42 @@ CREATE TEMPORARY VIEW fmtTable(fmtField) AS SELECT * FROM VALUES ('invalidFormat
 SELECT to_binary('abc', fmtField) FROM fmtTable;
 -- Clean up
 DROP VIEW IF EXISTS fmtTable;
+-- luhn_check
+select luhn_check('4111111111111111');
+select luhn_check('4111111111111111    ');
+select luhn_check('5500000000000004');
+select luhn_check('340000000000009');
+select luhn_check('6011000000000004');
```

Comment:
Ideally the more tests the better, but we should also think about coverage. The test cases I can think of
1. a few basic cases
2. spaces in the beginning/middle/end
3. non-digits
4. empty string
5. null

Reply:
I added suggested test cases